15/umbraco-ui-builder/collections/list-views.md

```diff
@@ -1,103 +1,159 @@
 ---
-description: Configuring the list view of a collection in Umbraco UI Builder, the backoffice UI builder for Umbraco.
+description: Configuring the list view of a collection in Umbraco UI Builder.
 ---
 
 # List Views
```

Comment:
(Duplicate comment, sorry 😅 but found it applies here as well)

Is this still called List View though?
If I remember correctly, this is what changed name to Collection? 🤔 Might not have in UI Builder..

Reply:
Same reply as above.. I know it's a little confusing 🙈 but the Collections section in the doc first talks about the colelctions Data Type and then diverges into configuring the list view (UI appearance) of a collection (Data Type) 😅